docs/TRANSLATIONS.md

```diff
@@ -1,115 +1,114 @@
 # Translations
 
-## Help us to review/proofread translations
+## Overview
 
-You can join our [GitHub translation teams](https://github.com/orgs/organicmaps/teams/translations/teams),
-so any contributor can tag all teams (or a specific language team) to get help with the review.
+The project consists of multiple components, each with its own translation files. Translations are managed via [Weblate][weblate], which integrates with Git, syncs files, offers a user-friendly interface, and automatically submits PRs with updates.
```

Comment:
I think this doc should be targeted mostly to translators.
And they don't need the most of this confusing technical info (git, PRs, etc.) especially upfront.

So it lacks a brief introduction to Weblate and how to start translating (just register? no approval / permissions required?) and a brief overview of components and dependency between Android and iOS strings.

Technical details for the devs could follow then.

Reply:
Updated.